service/lib/agama/software/product_builder.rb

```diff
@@ -45,13 +50,19 @@ def build
       # @return [Agama::Config]
       attr_reader :config
 
-      def create_product(id, data, attrs)
+      def create_product(id, data, attrs, cmdline_args)
         Agama::Software::Product.new(id).tap do |product|
           product.display_name = attrs["name"]
           product.description = attrs["description"]
           product.name = data[:name]
           product.version = data[:version]
-          product.repositories = data[:repositories]
+          # Issue#1474 Use agama.install_url=https://path-to-repodata
+          if cmdline_args.data.key?("install_url")
+            @logger.info "agama.install_url is set to " + cmdline_args.data["install_url"]
+            product.repositories = [cmdline_args.data["install_url"]]
```

Comment:
in general looks good, but this allows only single installation repository, which I am not sure is enough. Will staging need more installation repositories? if so, then I will add there some `split(",")` to allow multiple repos.

Reply:
good point, will do! also any idea how to to transform the code so ABC score/rubocop is happy?